### PR TITLE
Support changing table name at migrating

### PR DESCRIPTION
**Pull request by azexcy**
Changes proposed in this pull request:
- Support changing table name at migrating
- Remove unused DistSQL

### CODE REVIEW

File: shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/migration/MigrationDataConsistencyChecker.java [hunk 1]
```diff
@@ -119,16 +123,16 @@ private Map<String, DataConsistencyCountCheckResult> checkCount() {
...
-            Future<Long> targetFuture = executor.submit(() -> count(targetDataSource, table, targetDataSource.getDatabaseType()));
+            Future<Long> sourceFuture = executor.submit(() -> count(sourceDataSource, sourceTableName, sourceDataSource.getDatabaseType()));
+            Future<Long> targetFuture = executor.submit(() -> count(targetDataSource, targetTableName, targetDataSource.getDatabaseType()));
             long sourceCount = sourceFuture.get();
             long targetCount = targetFuture.get();
             return new DataConsistencyCountCheckResult(sourceCount, targetCount);
         } catch (final InterruptedException | ExecutionException ex) {
-            throw new PipelineDataConsistencyCheckFailedException(String.format("Count check failed for table '%s'", table), ex);
+            throw new PipelineDataConsistencyCheckFailedException(String.format("Count check failed for table '%s'", sourceDataSource), ex);
         }
```
Comment (sandynz) [inline shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-core/src/main/java/org/apache/shardingsphere/data/pipeline/scenario/migration/MigrationDataConsistencyChecker.java:136]: 1, Looks `sourceDataSource` should be `sourceTableName`

2, Maybe `targetTableName` should be added too. Is it necessary to use 2 `try ... catch`?

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/scenario/general/postgresql.xml [hunk 2]
```diff
@@ -56,19 +56,11 @@
...
         INSERT INTO test.t_order_item(item_id,order_id,user_id,status) VALUES(?,?,?,?)
     </full-insert-order-item>
-
-    <update-table-order-status>
-        UPDATE test.t_order SET status= 'unlock'
-    </update-table-order-status>
-
-    <create-index-status>
-        CREATE INDEX "idx_user_status" ON test.t_order ( status )
-    </create-index-status>
```
Comment (sandynz) [inline shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/scenario/general/postgresql.xml:73]: Is `CREATE INDEX` not need to test any more?

**Review by sandynz — CHANGES_REQUESTED** [review]

File: shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/scenario/general/postgresql.xml (hunk 2 above)
Comment (azexcy) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/scenario/general/postgresql.xml:73]: Yes, the index is used for stop source writing, no need any more

Comment (sandynz) [reply shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/resources/env/scenario/general/postgresql.xml:73]: OK

**Review by sandynz — APPROVED** [review]

**Pull request merged**